3-UsingAPIs/Week2/homework/ex1-programmerFun/index.js

```diff
@@ -17,27 +17,29 @@
 ------------------------------------------------------------------------------*/
 function requestData(url) {
   // TODO return a promise using `fetch()`
+  return fetch(url).then((response) => response.json());
 }
-
 function renderImage(data) {
   // TODO render the image to the DOM
+  const image = document.createElement('img');
+  image.src = data.img;
+  document.body.append(image);
   console.log(data);
 }
-
 function renderError(error) {
   // TODO render the error to the DOM
+  const errorMessage = document.createElement('H1');
```

Comment:
It is customary to use lowercase letters for HTML elements.

```suggestion
  const errorMessage = document.createElement('h1');
```

Reply:
Implemented